agent/statemanager/testdata/v17/secrets/ecs_agent_data.json

```diff
@@ -211,5 +215,5 @@
       "IPToTask": {}
     }
   },
-  "Version": 16
-}
\ No newline at end of file
+  "Version": 17
```

Comment:
isnt this 18?

Reply:
This is for secret integrating with ssm. I haven't add test cases for secrets integrating with asm since the change hasn't been merged into dev yet.